app/src/main/java/org/mifos/mobilebanking/models/User.kt

```diff
@@ -11,6 +12,7 @@ data class User (
 
     var userId: Long = 0,
     var isAuthenticated: Boolean = false,
+    @SerializedName(value = "username")
```

Comment:
@ethan627hsu It's also a correct solution. But Instead, rename `userName` to `username`.

Reply:
My solution has an advantage over the brute force, I don't have to refactor getters and setters across different files. My solution causes the least amount of overlap with different files.

Reply:
I know that, but it's just a typing mistake.